## Who is John?
**I'm a community-oriented, interdisciplinary scientist with a passion for studying and building resilient systems.**


**Questions that interest me:**

- How do biological systems respond to stress?
- How do we infer causality in high-dimensional and compositional biological datasets (-omics data)?
- How should we collect and efficiently process terabytes of relevant genetic sequencing data on a regular basis?

**Fields/tools I like:**

- statistical ecology and genetics
- stress physiology
- causal inference
- ontologies and knowledge graphs
- sometimes machine learning (when the time is right)


**What I do:** 

I build software for **efficient bioinformatic processing** and **biologically-inspired statistical modeling** of high-dimensional and compositional "-omics" data (e.g., metagenomics, metatranscriptomics, metabolomics). These projects often focus on **ecology of the gut microbiome**, **host-microbe interactions** and the **gut-brain axis**. 

I gravitate toward **collaborative projects** (where I can help someone else develop the best tools to answer their biological questions) and **mentoring/teaching**, as I see these as two of the most direct ways I can do good.



**In other contexts:**

From a non-professional perspective, I like to explore the areas around me, particularly via ultra-distance trail running and cycling. Music is a large part of my life, as I have played a variety of instruments (with a focus on trombone and banjo) for many years. I'm passionate about building community and support networks for myself and those around me, which I often find myself doing through cooking, music, and sport.


## Professional activities
I'm a **PhD candidate in the Interdisciplinary Quantitative Biology** program at the University of Colorado Boulder, housed by the Department of Integrative Physiology. I'm co-advised by Cathy Lozupone and Chris Lowry.

I'm interested in microbial community ecology, host-microbiome interactions, and drivers of resilience (both in microbial communities and in humans).

My research focuses on gut microbiome **multi-omics integration**, particularly host-microbiome dual transcriptomics data, with applications for studying the role of Helicobacter species as pathobionts influencing the development of colitis following psychosocial stress. I'm very interested in **causal inference in high-dimensional datasets**, compositional data analysis, biologically-informed statistical modeling, and sometimes machine learning if it's the appropriate tool for the job. 

## Background
I grew up in rural Northwest Tennessee, the land of corn and soy. I recieved my BS in Nutrition at East Tennessee State University, despite nearly majoring in music. At ETSU, I worked in Dr. Andy Clark's Nutritional Biochemistry Lab, which was my introduction to microbiome research.

In 2020, I moved west toward Boulder, Colorado, to join Chris Lowry's Behavioral Neuroendocrinology Lab at CU Boulder. Initially motivated by the Lowry Lab's work on whole body hyperthermia as an alternative treatment for major depressive disorder, I was very excited to study the impacts systemic physiology on behavior. Here, I found a passion for bioinformatics, microbial ecology, and studying the effects of microbial exposure on mood and stress-related disorders. 

In 2021, I joined the Interdisciplinary Quantitative Biology (IQ Bio) program via the BioFrontiers Institute. I was fortunate enough to spend the '21-'22 academic year rotating with Cathy Lozupone, Noah Fierer, and Nichole Reisdorph, where I worked on projects regarding microbiome co-occurrence networks, identifying metabolic pathways in metagenomic datasets, and integrating metabolomics and microbiome data. This year was fundamental to my computer science education and development as an interdisciplinary scientist, so I'm very grateful for everything that Kristin Powell and the IQ Bio team have done for me.

I also completed a team rotation project co-advised by Luke Evans and Maggie Stanislawski, where we constructed multi-omic risk scores to predict inflammatory bowel disease diagnosis with a transparent and interpretable framework using data from the Human Microbiome Project 2.

# Current projects
- [HoMi](https://github.com/sterrettJD/HoMi/): a pipeline for host-microbiome dual transcriptomics data (or mapping-based metagenomics/metatranscriptomics). HoMi is built atop Snakemake and manages data processing steps, software environments, and cluster job scheduling. 
- [Modulation](/GT-micro-metabo.pptx) of green tea compound absorption and neurotransmitter concentrations by the gut microbiome in gnotobiotic mice. This project utilizes novel food specific-omics approaches to track the concentrations of green tea-derived compounds in plasma of mice with humanized microbiomes. In collaboration between the Lozupone and Reisdorph labs.
- CauDA: a toolkit for implementing causal inference approaches in microbiome differential abundance analysis. In collaboration with Thomaz Bastiaanssen.

# Recent projects
- [Poly-omic risk scores](https://www.biorxiv.org/content/10.1101/2022.09.14.508056v1) predict inflammatory bowel disease diagnosis (IQ Bio team rotation with Luke Evans, Maggie Stanislawski, and Chris Gignoux; other students include Chris Arehart, Ruth Quispe-Pilco, and Rosie Garris)
- [SCNIC](https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13704): Sparse Correlation Network Analysis for Compositional Data (IQ Bio rotation with Cathy Lozupone)
- [A Metagenomic Investigation](https://journals.asm.org/doi/10.1128/msystems.00651-22) of Spatial and Temporal Changes in Sewage Microbiomes across a University Campus
- [The Influence of the Microbiota on Brain Structure and Function](https://link.springer.com/chapter/10.1007/978-3-030-91051-8_10): Implications for Stress-Related Neuropsychiatric Disorders - A review chapter in Evolution, Biodiversity and a Reassessment of the Hygiene Hypothesis
- [Interactive Microbial Ecology Exploration](https://sterrettjd.github.io/CSCI-6118-Project/): Jupyter Book for a Python tutorial on investigating microbial communities
- [Nicotine](/Nicotine-microbiome-rotation-talk.pptx) metabolism genes in the oral metagenomes of nicotine users vs non-users (IQ Bio rotation with Noah Fierer)
- [Characterization](https://www.wjgnet.com/1007-9327/full/v27/i33/5575.htm) of gut microbiome and metabolome in Helicobacter pylori patients in an underprivileged community in the United States